src/setup/internal.ts

```diff
@@ -1,6 +1,16 @@
-const setup = {
-  ipfs: {}
+export type Endpoints = {
+  api: string
+  lobby: string
+  user: string
 }
 
 
-export default setup
+export const setup = {
```

Comment:
should this be `let` since it's mutable?

Reply:
I think `const` is correct, since we don't want the top reference to change (which `let` would allow). So, in other words, `const` allows `setup.key = "change"`, but doesn't allow `setup = otherObject`. Which is what we want I think?

Reply:
ah yup you're right :+1:

Reply:
misread the setup methods & thought you were updating the top reference